index.html

```diff
@@ -6,21 +6,31 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <title>Awesome Books</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Inter&family=Poppins:wght@100&display=swap');
+    </style>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+    </style>
```

Comment:
- [OPTIONAL] The @import of fonts can be done in CSS file, I think it'll be good to have all styles in the external stylesheet.